terra-jupyter-r/Dockerfile

```diff
@@ -144,9 +151,9 @@ RUN R -e 'install.packages("BiocManager")' \
     "tidyverse", \
     "pbdZMQ", \
     "uuid"))' \
-    && chown -R $USER:users /home/jupyter-user/.local
+		&& R -e 'BiocManager::install("DataBiosphere/Ronaldo")'
```

Comment:
Can `Ronaldo` remain in the array above?

Reply:
no..for some reason it doesn't install properly when it's installed with `BiocManager::install(c(??))` (notice that `c`)